x.py

```diff
@@ -262,11 +285,13 @@ def test_go(dir: str, rest: List[str]) -> None:
         help="Check or lint source code")
     parser_check.set_defaults(func=parser_check.print_help)
     parser_check_subparsers = parser_check.add_subparsers()
-    parser_check_cpplint = parser_check_subparsers.add_parser(
-        'cpplint',
-        description="Lint code with cpplint (https://github.com/cpplint/cpplint)",
-        help="Lint code with cpplint (https://github.com/cpplint/cpplint)")
-    parser_check_cpplint.set_defaults(func=cpplint)
+    parser_check_format = parser_check_subparsers.add_parser(
+        'format',
+        description="Format source code",
+        help="Format source code")
+    parser_check_format.set_defaults(func=check_format)
+    parser_check_format.add_argument('--clang-format-path', default='clang-format', help="path of clang-format used to check source")
+    parser_check_format.add_argument('-i', default=False, action='store_true', help="format source files")
```

Comment:
A check should not change files. Perhaps you add both `./x.py check clang-format` and `./x.py fix clang-format`.

Reply:
done